queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * from animals WHERE NAME LIKE '%mon%';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT* FROM animals WHERE NEUTERED = TRUE AND ESCAPE_ATTEMPTS < 3;
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT * FROM animals WHERE NAME IN ('Agumon', 'Pikachu');
+
+-- List name and escape attempts of animals that weigh more than 10.5kg.
+SELECT NAME, ESCAPE_ATTEMPTS FROM animals WHERE weight_kg > 10.5;
+
+-- Find all animals that are neutered.
+SELECT * FROM animals WHERE NEUTERED = TRUE;
+
+-- Find all animals not named Gabumon.
+SELECT * FROM animals WHERE NAME IN ('Gabumon');
```

Comment:
- [ ] Kindly fix this query as in this instance you are actully selecting all records with the name `Gabumon`